nautobot/core/celery/schedulers.py

```diff
@@ -127,6 +127,11 @@ def apply_async(self, entry, producer=None, advance=True, **kwargs):
             if task:
                 scheduled_job = entry.model
                 job_queue = scheduled_job.job_queue
+                if job_queue is None:
+                    job_queue, _ = JobQueue.objects.get_or_create(
+                        name=settings.CELERY_TASK_DEFAULT_QUEUE,
+                        defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY},
+                    )
                 # Distinguish between Celery and Kubernetes job queues
                 if job_queue.queue_type == JobQueueTypeChoices.TYPE_KUBERNETES:
```

Comment:
Is this the right fix? We only use the `job_queue` here if it's a k8s queue. Would a better fix be:

```suggestion
                # Distinguish between Celery and Kubernetes job queues
                if job_queue is not None and job_queue.queue_type == JobQueueTypeChoices.TYPE_KUBERNETES:
```